src/test/java/com/cloudbees/jenkins/support/SupportTestUtils.java

```diff
@@ -276,7 +277,11 @@ public static void testPermissionToSeeAction(JenkinsRule j,
             HtmlPage page = wc.goTo(baseUrl);
             HtmlDivision sidePanel = (HtmlDivision) page.getElementById("side-panel");
             assertEquals(userPrivileged + " should be able to see the Support action", 1,
-                sidePanel.getElementsByAttribute("a", "title", action.getDisplayName()).size());
```

Comment:
title attribute is no longer set, just iterate through instead, could switch to xpath too but this is quite straightforward